celest/coordinates/astronomical_quantities.py

```diff
@@ -0,0 +1,602 @@
+
+
+from celest.units.quantity import Quantity
+from celest import units as u
+from celest.constants import (
+    JD2000_DATE,
+    DAYS_IN_JULIAN_CENTURY,
+    EARTH_ROTATION_RATE_DEG_PER_DAY,
+    ERA_AT_JD2000_DEG
+)
+from typing import Tuple
+import numpy as np
+
+
+def earth_rotation_angle(julian: Quantity) -> Quantity:
+    """Return Earth rotation angle in decimal degrees.
+
+    Parameters
+    ----------
+    julian : Quantity
+        Quantity object containing date data in the JD2000 epoch.
+
+    Returns
+    -------
+    Quantity
+        Earth rotation angle in decimal degrees.
```

Comment:
The return statement doesn't have to be degrees since it returns a `Quantity` object. Update all return documentation to reflect this.